qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -230,7 +242,7 @@ def get_project_name(self) -> str:
     def get_limit(self, limit: Optional[int]) -> int:
         """Get the allowed limit on results to return in queries based on the
          given `limit`. If this number is greater than the
-         mongoengine_soket.max_limit then the max_limit will be returned instead.
+         mongoengine_socket.max_limit then the max_limit will be returned instead.
```

Comment:
Docstring change: `SQLAlchemy_socket`. There are some other references to mongoengine in this file, but all as commented out lines

Reply:
Got it.